### DISCUSSION TITLE
Change inverter option

### DISCUSSION THREAD

**Discussion by lucasimons** [softwarecrash/Solar2MQTT-ESP8266#2] [Ideas]
I use a rp4 with solpiplog but i want to use a esp8266 to send the mqtt value to home assistant, the solpiplog send this value 

masterstatus = Battery
status2 = aut
charge = 3
status = bat
acin = 230.0
acinhz = 50.0
acout = 228.0
acouthz = 50.0
acoutva = 0000
acoutw = 0000
load = 000
busv = 377
battv = 26.98
battchrg = 002
battcappa = 100
heatsinktemp = 0035
pvibatt = 0003
pvvolt = 032.2
battvscc = 26.95
batdischrg = 00000
devicestatus = 10010111
pvchargew = 00097
bulk = 28.2
float = 27.0

I can change the charging priority of my easun inverter this is possibile with this code?

**Comment by softwarecrash** [comment]
sorry, but i dont plane to add options to modify the inverter settings

**Comment by softwarecrash** [comment]
function added to full controll the data
